packages/react-components/src/components/chart/types.ts

```diff
@@ -68,7 +68,11 @@ export type ChartStyleSettings = {
   [refId: string]: ChartStyleSettingsOptions;
 };
 
-export type InternalGraphicComponentGroupOption = { timestampInMs: number } & {
+export type InternalGraphicComponentGroupOption = {
+  timestampInMs: number;
+  yAxisMarkerValue: number[];
+  headerColor: string;
+} & {
```

Comment:
is there a reason why this is an intersection and not one type?

Reply:
+1

Reply:
Ideally this should be just
 ```
GraphicComponentGroupOption & {timestamp : number , yAxisMarkerValue: number[];
  headerColor: string;}
```
the typescript is not happy about GraphicComponentGroupOption, and it throws error whenever we try to access the children.

the timestamp yAxisMarkerValue and headerColor is not part of the GraphicComponentGroupOption. these fields are introduced to access then easier in legend else we will have to parse the children / series objects